app/src/main/java/nu/yona/app/YonaApplication.java

```diff
@@ -202,5 +212,54 @@ public static Tracker getTracker()
 		}
 		return mContext.tracker;
 	}
+
+
+	private ServiceConnection getVpnServiceConnection()
+	{
+		return connection = new ServiceConnection()
```

Comment:
This is a weird construction. Why are you assigning that member here? You're only using it on line 229, right? Can't you simply use ``this`` there and do away with the member?